web/tests/lynxkite.ts

```diff
@@ -425,14 +416,8 @@ export class BoxEditor extends PopupBase {
     this.head().click(); // Make sure the parameters are not focused.
   }
 
-  expectParameter(paramName, expectedValue) {
-    const param = this.element.locator('div#param-' + paramName + ' input');
-    expect(param.getAttribute('value')).toBe(expectedValue);
-  }
-
-  expectSelectParameter(paramName, expectedValue) {
-    const param = this.element.locator('div#param-' + paramName + ' select');
-    expect(param.getAttribute('value')).toBe(expectedValue);
+  getParameter(paramName, tag = 'input') {
+    return this.element.locator(`div#param-${paramName} ${tag}`);
   }
```

Comment:
The old code had lots of functions that make assertions. I think if the assertion is simple enough, then it's easier to understand if it's made in the test. (Not in `lynxkite.ts`.)